src/routes/docs/floating/FloatingDefault.svelte

```diff
@@ -0,0 +1,15 @@
+<script>
+	import CardBody from '$lib/card/CardBody.svelte'
```

Comment:
@abdolian please update this import

`import { Card, CardBody, Dot, Floating } from 'yesvelte'`